doc/manual/embedding.rst

```diff
@@ -56,6 +56,10 @@ Before the program terminates, it is strongly recommended to call ``jl_atexit_ho
    >>> julia.jl_init('.')
    250593296
 
+.. note::
+
+ If the julia program needs to access symbols from the main executable, it may be necessary to add ``-Wl,--export-dynamic`` linker flag at compile time on Linux in addition to the ones generated by ``julia-config.jl`` described below. This is not necessary when compiling a shared library.
```

Comment:
This is an awkward way to link an executable (more standard way would be to provide a registration function so that the linker dependency graph can be a DAG), but I guess it happens.